src/Resolution.ts

```diff
@@ -378,17 +387,28 @@ export default class Resolution {
     return this.getNamingMethodOrThrow(domain).serviceName(domain);
   }
 
+  /**
+   * Returns all record keys of the domain.
+   * Currently supports only CNS
```

Comment:
You need to implement that for all naming services

```suggestion
   * This method is strongly unrecommended for production use due to lack of support for many ethereum service providers and low performance
   * Method is not supported by ENS
```